src/test/resources/requests/adaptivecard-mention.json

```diff
@@ -0,0 +1,72 @@
+{
+  "type": "message",
+  "attachments": [
+    {
+      "contentType": "application/vnd.microsoft.card.adaptive",
+      "content": {
+        "type": "AdaptiveCard",
+        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+        "version": "1.4",
+        "msteams": {
+          "width": "Full",
+          "entities": [
```

Comment:
if this is an array, lets add one more mention user to the list so we have multi elements in example